src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -610,7 +610,7 @@
 examples:
   - name: Show the storage blob delete-policy. (autogenerated)
     text: |
-        az storage blob service-properties delete-policy show --account-name mystorageccount --account-key 00000000
+        az storage blob service-properties delete-policy show --account-name mystorageccount
```

Comment:
`--account-key 00000` should also be useful.

Reply:
You can contact in Aladdin Collabration channel to report that.

Reply:
Shall I keep it?

Reply:
yes

Reply:
fixed